macos/FluentUI/Button/Button.swift

```diff
@@ -100,17 +100,15 @@ open class Button: NSButton {
 
 		if let title = title {
 			self.title = title
-			if let image = image {
-				self.image = image
-				self.imagePosition = imagePosition
-			} else {
-				self.imagePosition = .noImage
-			}
-		} else if let image = image {
+		} else {
+			/// NSButton defaults title to "Button"
```

Comment:
Nit. Three slashes is meant to be public documentation, while 2 is meant to be internal comments. I think this should be 2 slashes

Reply:
noted.